src/includes.test.ts

```diff
@@ -0,0 +1,126 @@
+import { pipe } from './pipe';
+import { includes } from './includes';
+
+describe('includes', () => {
+  describe('data first', () => {
+    it('works with number arrays', () => {
+      expect(includes([1, 2, 3, 4, 5], 2)).toBe(true);
+      expect(includes([1, 2, 3, 4, 5], 6)).toBe(false);
+    });
+
+    it('works with string arrays', () => {
+      expect(includes(['a', 'b', 'c', 'd', 'e'], 'c')).toBe(true);
+      expect(includes(['a', 'b', 'c', 'd', 'e'], 'f')).toBe(false);
+    });
+
+    it('works with objects', () => {
+      expect(includes({ a: 1, b: 2, c: 3 }, 'a')).toBe(true);
```

Comment:
This is not how lodash `includes` works. It checks for value, not for a key.

Reply:
Ok I see, so checking keys would rather be the job of `has`?

Reply:
Now checking object values